### PR TITLE
added preprint citation info

### PR DESCRIPTION
**Pull request by nbokulich**

### CODE REVIEW

**Review by thermokarst — APPROVED** [review]
MVP 2017.

File: q2_longitudinal/plugin_setup.py [hunk 1]
```diff
@@ -31,7 +31,13 @@
         'This QIIME 2 plugin supports methods for analysis of time series '
         'analysis, involving either paired sample comparisons or longitudinal '
         'study designs.'),
-    short_description='Plugin for paired sample and time series analyses.'
+    short_description='Plugin for paired sample and time series analyses.',
+    citation_text=("q2-longitudinal: a QIIME 2 plugin for longitudinal and "
+                   "paired-sample analyses of microbiome data. Nicholas "
+                   "Bokulich, Yilong Zhang, Matthew Dillon, Jai Ram Rideout, "
+                   "Evan Bolyen, Huilin Li, Paul Albert, James Gregory "
```
Comment (gregcaporaso) [inline q2_longitudinal/plugin_setup.py]: Could you change _James_ to _J_?

Comment (nbokulich) [reply q2_longitudinal/plugin_setup.py]: 👍 sorry — was just copy/pasting the author list from biorxiv.

fixed.

Comment (gregcaporaso) [reply q2_longitudinal/plugin_setup.py]: Thanks!

**Pull request merged**